### DISCUSSION TITLE
Module '"@headlessui/vue"' has no exported member(s) 'TabGroup, TabList, Tab, TabPanels, TabPanel''

### DISCUSSION THREAD

**Discussion by nkosi-tauro** [tailwindlabs/headlessui#739] [Help]
Using version 1.4.0 and following the example in the [documentation](https://headlessui.dev/vue/tabs).
`import { TabGroup, TabList, Tab, TabPanels, TabPanel } from '@headlessui/vue';`

**Comment by diegosanw** [comment]
Same issue here. It's a vetur warning, only when using TypeScript.

Solved it by adding 
`declare module "@headlessui/vue";`
to typings.d.ts

It's odd that any other headless component works fine without this line though.
